src/logs/fatal_on.go

```diff
@@ -0,0 +1,46 @@
+package logs
```

Comment:
Thoughts on putting this in the util package under something like `util.ExitOn` and `util.ExitOnf`

Reply:
Ultimately I want to extract this out of this code base into a stand-alone library. The purpose is to make `log.Fatal` available without repetitive `if err != nil` boilerplate, without deviating too much from the origital. I think `logs.FatalOn` is as close to the original interface as it gets.